.travis.yml

```diff
@@ -24,8 +24,11 @@ jobs:
         echo "Running visual difference tests...";
         npm run test:diff || travis_terminate 1;
       fi
-  - stage: update-version
-    script: frauci-update-version -d=skip && export TRAVIS_TAG=$(frauci-get-version)
+after_success:
+- |
+  if [ $TRAVIS_BUILD_STAGE_NAME == "Code-tests" ]; then
```

Comment:
I think we will want to wait until after all are successful.  It looks like this change would only wait until after the first stage.

Reply:
If we wait until all are successful, we have the issue where the version gets bumped twice, we had this issue on the `activities` repo.

Reply:
We could change the if to `$TRAVIS_BUILD_STAGE_NAME == "Visual-difference-tests"`. That way it would bump version (if requested) after the vdiff tests pass, which will only run after `Code-tests`.

Reply:
https://github.com/BrightspaceUI/core/pull/251

Reply:
That would be _better_.  The bulk of our tests are visual-diff, so only waiting for the unit tests is a significant exposure.

Reply:
Yes that was my bad, I didn't realize that my change would bump after unit tests. This will wait for unit tests to finish before running vdiff and only bump if vdiff passes.

Reply:
And only bump if requested in the commit message